css/style.css

```diff
@@ -379,6 +652,59 @@
   .data__speed-block {
     padding-right: 55px;
   }
+
+  .data__figures {
+    font-weight: 600;
+}
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)